recipe/meta.yaml

```diff
@@ -10,32 +10,45 @@ source:
   sha256: dd5be4a72d8a2995c3f583cf82bf3cd1a9544cfdabf2d22595b67aff07349666
 
 build:
-  noarch: python
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 1
+  number: 0
+  skip: True  # [py<36]
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - jsonpath
```

Comment:
why is `jsonpath` here? 

I start to believe that this is just this `jsonpath` https://github.com/sean2077/jsonpath-python/blob/aead1e4db9890f12fc029ca57b9732aa16c62915/jsonpath/__init__.py

and NOT another dependency.

conda-forge and us probably got it wrong? @ifitchet @JeanChristopheMorinPerso

Reply:
I think so yes. jsonpath-python uses a flat layout instead of a src layout. So `jsonpath` will be importable from the current directory (and so setup.py).

Reply:
Yes, it appears to work and pass its tests without the `jsonpath` from elsewhere.